### PR TITLE
Update tutorial_tn_circuits.py

### PR DESCRIPTION
**Pull request by KetpuntoG**
Adding affiliation BMW to Tensor network demo

### CODE REVIEW

**Review by DSGuala — APPROVED** [review]
Do we have to change any files to make sure the "Last updated" date is correct? Otherwise looks good to me :+1:

**Comment by KetpuntoG** [comment]
> Do we have to change any files to make sure the "Last updated" date is correct? Otherwise looks good to me 👍

Let's do that just in significant changes :)

File: demonstrations/tutorial_tn_circuits.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
 | :sup:`1` Xanadu, Toronto, ON, M5G 2C8, Canada
 | :sup:`2` BMW Group, Munich, Germany
```
Comment (DSGuala) [inline demonstrations/tutorial_tn_circuits.py:19]: I still see these in the preview. Should they be commented out based on this [discussion](https://xanaduhq.slack.com/archives/C05C2NU8QBY/p1705074522695049?thread_ts=1705073042.243859&cid=C05C2NU8QBY)?

File: demonstrations/tutorial_tn_circuits.py [hunk 2]
```diff
@@ -15,10 +15,7 @@
 *Authors: Diego Guala*:superscript:`1` *, Esther Cruz-Rico*:superscript:`2` *,
 Shaoming Zhang*:superscript:`2` *, Juan Miguel Arrazola*:superscript:`1` *— Posted: 29 March 2022. Last updated: 27 June 2022.*
```
Comment (ikurecic) [inline demonstrations/tutorial_tn_circuits.py:16]:
```suggestion
*Authors: Diego Guala*:superscript:`1` *, Esther Cruz-Rico*:superscript:`2` *,
Shaoming Zhang*:superscript:`2` *, Juan Miguel Arrazola*:superscript:`1` *— Posted: 29 March 2022. Last updated: 27 June 2022.*
```
```suggestion
*Authors: Diego Guala, Esther Cruz-Rico,
Shaoming Zhang, Juan Miguel Arrazola — Posted: 29 March 2022. Last updated: 27 June 2022.*
```

**Review by ikurecic — COMMENTED** [review]
Maybe just remove the superscripts altogether?

File: demonstrations/tutorial_tn_circuits.py (hunk 2 above)
Comment (KetpuntoG) [reply demonstrations/tutorial_tn_circuits.py:16]: I'll delete this in another PR that I am doing where I'm removing all the authors and metadata. We can leave it like this here :)

**Review by ikurecic — APPROVED** [review]
kk

**Pull request merged**